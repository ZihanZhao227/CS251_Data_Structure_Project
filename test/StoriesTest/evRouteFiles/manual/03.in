3
0 0 0 CITY Start
1 20 0 CHARGING_STATION CS1
2 40 0 CITY End
