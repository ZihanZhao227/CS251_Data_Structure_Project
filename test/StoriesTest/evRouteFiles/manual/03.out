0 2 100.000 40.000
0 2
